lib/puma/cluster.rb

```diff
@@ -22,7 +23,9 @@ def initialize(launcher)
       @workers = []
       @next_check = Time.now
 
-      @phased_restart = false
+      @pending_phased_restart = false
```

Comment:
renamed this variable to make flow a little clearer; it no longer overlaps with a method name and contrasts more clearly with `in_phased_restart` below

Reply:
Extracted into https://github.com/puma/puma/pull/3645.